cmd/cli/user.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"github.com/moira-alert/moira"
+)
+
+func transferUserSubscriptionsAndContacts(database moira.Database) error {
+	if *fromUser == "" && *toUser == "" {
+		return nil
+	}
+
+	contactIDs, err := database.GetUserContactIDs(*fromUser)
+	if err != nil {
+		return err
+	}
+	contacts, err := database.GetContacts(contactIDs)
+	if err != nil {
+		return err
+	}
+	for _, contact := range contacts {
+		contact.User = *toUser
+		if err = database.SaveContact(contact); err != nil {
+			return err
+		}
+	}
+
+	subscriptionIDs, err := database.GetUserSubscriptionIDs(*userDel)
+	if err != nil {
+		return err
+	}
+	subscriptions, err := database.GetSubscriptions(subscriptionIDs)
+	if err != nil {
+		return err
+	}
+	for _, subscription := range subscriptions {
+		subscription.User = *toUser
+	}
+	return database.SaveSubscriptions(subscriptions)
+}
+
+func deleteUser(database moira.Database) error {
```

Comment:
Same comment as earlier. I will be more readable if we will receive user id here via function arguments.